content/faq/android-lags.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "F.A.Q. - Android"
+description: "Frequently asked questions for Organic Maps application"
+weight: 1000
+
+taxonomies:
+  faq: ["Android"]
+
+extra:
+  question: Why OrganicMaps is slow on my device, UI is laggy, and it crashes randomly?
```

Comment:
It would be great to autogenerate slugs from the title/question, so users will have a readable links like /faq/android/why-organicmaps-is-slow-on-my-device/

Reply:
We don't have separate page for each question.